django/core/management/templates.py

```diff
@@ -54,6 +55,12 @@ def add_arguments(self, parser):
             help='The file name(s) to render. Separate multiple file names '
                  'with commas, or use -n multiple times.'
         )
+        parser.add_argument(
+            '--exclude', '-x', dest='exclusions',
+            action='append', default=argparse.SUPPRESS, nargs='?', const='',
+            help='The directory name(s) to exclude. Use -x multiple times '
+                 'to separate multiple directory names.'
```

Comment:
"to separate multiple directory names" -> "to exclude multiple directories"

Also, I think the first sentence should say, "The directory name(s) to exclude, in addition to .git and \_\_pycache__." I think this "in addition" clause should also be added to the documentation changes below.

Reply:
Okay, updated.